ovirt-imageio.spec.in

```diff
@@ -145,6 +172,53 @@ if systemctl is-active ovirt-imageio-daemon.service >/dev/null; then
     systemctl stop ovirt-imageio-daemon.service
 fi
 
+%if %{with_python38}
+%define python3_pkgversion 38
+%define __python3 /usr/bin/python3.8
+
+%package python38-common
+Summary:   oVirt imageio common resources
+
+# NOTE: keep in sync with automation/build-artifacts.packages
+BuildRequires: gcc
+BuildRequires: python38-devel
+
+# NOTE: keep in sync with automation/check.packages
+Requires:  python38
+
+%description python38-common
+Common resources used by oVirt imageio server and client
+
+%files python38-common
+%license COPYING
+%{python3_sitearch}/%{srcname}
+%{python3_sitearch}/%{srcname}-*.egg-info
+%exclude %{python3_sitearch}/%{srcname}/client
+%exclude %{python3_sitearch}/%{srcname}/admin
+
+
+%package python38-client
+Summary:   oVirt imageio client library
+
+# NOTE: keep in sync with automation/check.packages
+Requires:  python38-%{name}-common = %{version}-%{release}
```

Comment:
This requires the `python38-ovirt-imageio-common` but above it is build with name `ovirt-imageio-python38-common`, please change the requirement name.